doc/source/myrocks/zenfs.rst

```diff
@@ -1,125 +1,242 @@
+ZenFS depends on the ``libzbd`` user library and requires a Linux kernel implementation that supports NVMe Zoned Namespaces. The kernel
+must be configured with `zone block device support
+enabled <https://zonedstorage.io/docs/linux/config>`__.
 
+Read the `Western Digital and Percona deliver
+Utrastar DC ZN540 Zoned Namespace SSD support for Percona Server for
+MySQL <https://documents.westerndigital.com/content/dam/doc-library/en_us/assets/public/western-digital/collateral/company/western-digital-zns-ssd-perconal-blogpost.pdf>`__ PDF for more information.
 
-#. Run ``/zenfs mkfs --zbd <short_block_device_name> --aux_path=<path_to_aux_zenfs_dir>`` in the context of the mysql:mysql account to initialize ZenFS on the ZBD.
 
-    .. sourcecode:: bash
+The following procedure installs Percona Server for MySQL and then configures 
+``--rocksdb-fs-uri=zenfs://dev:<short_block_device_name>`` for data storage.
 
-        zenfs mkfs --zbd=nvme1n2 --aux_path=/var/lib/mysql_aux
+.. note::
 
-    .. note::
+   The ``<block_device_name>`` can have a short name designation which is the ``<short_block_device_name>``. For the purposes of this document, the ``block_device_name`` is ``/dev/nvme0n2`` and the short name is ``nvme0n2``. 
+    
+For the moment, Percona provides a binary release for `Debian 11.1 <https://www.debian.org/releases/bullseye/debian-installer/index>`__. `Other Linux distributions <https://zonedstorage.io/docs/distributions/linux/>`__ are adding support for ZenFS, but Percona does not provide installation packages for those distributions yet.  
 
-        This step initializes the metadata on the block device, ``<block_device_name>``, and creates the ``<path_to_aux_zenfs_dir>`` owned by the `mysql:mysql` account. For example, the ``<block_device_name>`` could be ``/dev/nvme1n2``. The ``<path_to_aux_zenfs_dir>`` is a unique, dedicated directory which allows read and write access to `mysql:mysql`.
 
-        If you are re-initializing an already initialized ZBD, you must manually clear the ``/var/lib/mysql_aux`` directory and add the additional ``--force`` parameter.
+.. _zenfs-installation:
 
-        .. sourcecode:: bash
+Installation
+============================================================
 
-            zenfs mkfs --zbd=nvme1n2 --aux_path=/var/lib/mysql_aux  --force
+Start with the installation of *Percona Server for MySQL*. 
 
+1. The steps are listed here for convenience, for an explanation, see :ref:`apt-install`.
 
 
-#. You can start Percona Server for MySQL with any of the known methods. For example, add this option to ``my.cnf``:
+   .. code-block:: bash
 
-    .. sourcecode:: text
+      $ sudo apt install gnupg2	
+      $ wget https://repo.percona.com/apt/percona-release_latest.$(lsb_release -sc)_all.deb	
+      $ sudo dpkg -i percona-release_latest.$(lsb_release -sc)_all.deb	
+      $ sudo percona-release setup ps80
```

Comment:
These steps can be slightly simplified (I'm working on updating the generic percona-release docs accordingly):
```suggestion
      $ wget https://repo.percona.com/apt/percona-release_latest.generic_all.deb	
      $ sudo apt install gnupg2 lsb-release ./percona-release_latest.generic_all.deb	
      $ sudo percona-release setup ps80
```